merkletree/proof.go

```diff
@@ -88,13 +89,22 @@ func VerifyAuthPath(ap *AuthenticationPath,
 	return true
 }
 
+func VerifyCommitment(salt []byte, key string, value []byte, commitment []byte) bool {
+	got := crypto.Digest(salt, []byte(key), value)
```

Comment:
Since it was fine in that moment. I think we should add `crypto.Commitment` back, for now.